app/views/tasks/_task_table.html.erb

```diff
@@ -0,0 +1,60 @@
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Description
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <% if @task.completed_date.present? %>
+      <th>Completion Date</th>
+    <% end %>
+    <% if current_page?(action: 'show') %>
```

Comment:
The second or third time I wrote it I knew I should be abstracting it away, but wasn't sure what exactly that would look like in terms of the code. Thanks!